OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -510,18 +517,21 @@ static void SetSDLAttributes(GLProfile profile)
 			}
 		}
 
-		static bool CanCreateGLWindow(GLProfile profile)
+		bool CanCreateGLWindow(GLProfile profile)
```

Comment:
IMO it would be cleaner to keep this static and pass the error list in as an argument. That can then be defined as a local variable within the ctor.